utils/outputwriter/outputcontent.go

```diff
@@ -118,288 +157,719 @@ func footer(writer OutputWriter) string {
-		dependencies = append(dependencies, fmt.Sprintf(format, component.Name, component.Version))
+// License violations
+
+func getLicenseViolationsContent(issues issues.ScansIssuesCollection, writer OutputWriter) (content []string) {
+	if len(issues.LicensesViolations) == 0 {
+		return []string{}
 	}
-	return
+	content = append(content, getLicenseViolationsSummaryTable(issues.LicensesViolations, writer))
+	content = append(content, getLicenseViolationsDetailsContent(issues.LicensesViolations, writer)...)
+	return ConvertContentToComments(content, writer, getDecoratorWithLicenseViolationTitle(writer))
 }
 
-func getCveIdsCellData(cveRows []formats.CveRow) (ids CellData) {
-	if len(cveRows) == 0 {
-		return NewCellData()
-	}
-	for _, cve := range cveRows {
-		ids = append(ids, cve.Id)
+func getDecoratorWithLicenseViolationTitle(writer OutputWriter) func(int, string) string {
+	return func(commentCount int, content string) string {
+		contentBuilder := strings.Builder{}
+		// Decorate each part of the split content with a title as prefix and return the content
+		WriteContent(&contentBuilder, writer.MarkAsTitle(licenseViolationTitle, 3))
+		WriteContent(&contentBuilder, content)
+		return contentBuilder.String()
 	}
-	return
 }
 
-type vulnerabilityOrViolationDetails struct {
-	details           string
-	title             string
-	dependencyName    string
-	dependencyVersion string
+func getLicenseViolationsSummaryTable(licenses []formats.LicenseViolationRow, writer OutputWriter) string {
+	table := NewMarkdownTable("Severity", "License", "Direct Dependencies", "Impacted Dependency", "Watch Name").SetDelimiter(writer.Separator())
+	if _, ok := writer.(*SimplifiedOutput); ok {
+		// The values in this cell can be potentially large, since SimplifiedOutput does not support tags, we need to show each value in a separate row.
+		// It means that the first row will show the full details, and the following rows will show only the direct dependency.
+		// It makes it easier to read the table and less crowded with text in a single cell that could be potentially large.
+		table.GetColumnInfo("Direct Dependencies").ColumnType = MultiRowColumn
+	}
+	for _, license := range licenses {
+		table.AddRowWithCellData(
+			NewCellData(writer.FormattedSeverity(license.Severity, "Applicable")),
+			NewCellData(license.LicenseKey),
+			getDirectDependenciesCellData(license.Components),
+			NewCellData(fmt.Sprintf("%s:%s", license.ImpactedDependencyName, license.ImpactedDependencyVersion)),
+			NewCellData(license.Watch),
+		)
+	}
+	return writer.MarkInCenter(table.Build())
 }
 
-func vulnerabilityDetailsContent(vulnerabilities []formats.VulnerabilityOrViolationRow, writer OutputWriter) (content []string) {
-	vulnerabilitiesWithDetails := getVulnerabilityWithDetails(vulnerabilities)
-	if len(vulnerabilitiesWithDetails) == 0 {
+func getLicenseViolationsDetailsContent(licenseViolations []formats.LicenseViolationRow, writer OutputWriter) (content []string) {
```

Comment:
same as getScaSecurityIssueDetailsContent comment - the way this func works is a bit odd since we iterate in a loop and acts according to the entire slice length inside the loop